torchtune/utils/metric_logging.py

```diff
@@ -317,3 +317,127 @@ def close(self) -> None:
         if self._writer:
             self._writer.close()
             self._writer = None
+
+
+class CometLogger(MetricLoggerInterface):
+    """Logger for use w/ Comet (https://www.comet.com/site/).
```

Comment:
Could you add a quick sentence describing what Comet is? (I know we didn't have that for the WandBLogger)

Reply:
@Lothiraldan 

"Comet is an experiment tracking tool that helps ML teams track, debug, compare, and reproduce their model training runs"